### ISSUE TITLE
Broken link after Turbo Stream render

### ISSUE THREAD

**Issue by dfmoreto** [hotwired/turbo-rails#213]
When I render a TurboStream view to replace a content, the link inside it doesn't work anymore.

I currently have this **update** action in my controller that updates a User
```
def update
    @user = User.find(params[:id])
    @user.attributes = user_params
    if @user.save
      render :update
    else
      render :edit, status: :unprocessable_entity
    end
  end
```
This action renders this **update.turbo_stream.erb**
```
<%= turbo_stream.replace dom_id(@user) do %>
  <%= render "table_row", user: @user %>
<% end %>
```
And this partial `table_row` has this content
```
<tr id="<%= dom_id(user) %>">
  <td><%= user.name %></td>
  <td><%= user.email %></td>
  <td><%= user.enum_text(:profile) %></td>
  <td><%= user.enum_text(:plan) %></td>
  <td>
    <div class="buttons are-small">
        <%= link_to t('.edit'), edit_user_path(user), data: { turbo_frame: 'user-modal-content' },
                    class: "button is-text has-text-info" %>
      <%= button_to t('.remove'), user, method: :delete, data: { confirm: t('.confirm'), turbo: false },
                  class: "button is-text has-text-danger" %>
    </div>
  </td>
</tr>
```
This partial is also rendered in a Users table and when a User is updated, TurboStream also updates the table line with the new content row.
The problem is: after the TurboStream rendering the Edit link on this partial doesn't work anymore.
Am I missing something?

**Comment by dhh** [comment]
Possible that the "turbo: false" part is not picked up by a mistake after replacing? If you can reproduce that outside your app, please try to do so. Also, you could convert that button_to to use Turbo instead of UJS and see if that helps.

**Comment by dfmoreto** [comment]
I reproduced it outside my solution and I had a bit different problem, but may have the same origin as my previous problem.

I've built a very simple User app in this repo using hotwire-rails: 
https://github.com/dfmoreto/turbo-sample

In this app, I have this "New User" link on `users/index.html.erb` with a TurboFrame just below it
```
...
<%= link_to 'New User', new_user_path, data: { turbo_frame: "form-area" } %>

<%= turbo_frame_tag "form-area" do %>
<% end %>
``` 
There is a table with and ID "users" in `tbody`, but I hided from the snipped.

Clicking on New User, my form is rendered inside this `turbo_frame_tag` using this view `new.html.erb`
```
<%= turbo_frame_tag "form-area" do %>
  <%= render 'form', user: @user %>
<% end %>
```

Submitting the form, a new User is created and a `create.turbo_stream.erb` view is rendered with this content
```
<%= turbo_stream.append "users" do %>
  <%= render "table_row", user: @user %>
<% end %>

<%= turbo_stream.replace "form-area" do %>
<% end %>
```

But after I submit a New User, or even try to edit a new one, it seems to be that Turbo stop working because all links starts to render full page instead of only replacing contents where it should. They only work again if I reload the page.

Tell me if more details are needed.

**Comment by dfmoreto** [comment]
I've just discovered my mistake on the sample app I sent before. I was rendering a TurboStream to replace a TurboFrame without rerendering TurboFrame with it. It was why in the first attempt I rendered the form and on the second I was "stopping to work".

I'll try to discover the same on my original app. I also found workaround in some place I don't remember: if I add a random parameter on my `link_to` route as below, it works after a TurboStream replace.
```
<%= link_to t('.edit'), edit_user_path(user, t: Time.now.to_i), data: { turbo_frame: 'user-modal-content' },
                    class: "button is-text has-text-info" %>
```
